src/cmd/link/internal/ld/deadcode.go

```diff
@@ -90,6 +90,11 @@ func (d *deadcodePass) init() {
 			first, last = -14, 479
 		case sys.RISCV64:
 			first, last = 1, 1023
+		case sys.MIPS64:
+			first, last = 1, 5
+			// runtime.unhandledExternalInterrupt is also a handler.
+			// Must be called somewhere in the code, otherwise it
+			// gets removed without error and relocs point to 0x0.
```

Comment:
I added it as you recommended. Not sure about the other GOARCHs. It's probably safe because it is called in externalInterruptHandler anyways, but while I was implementing this I ran into this error so I added the comment.